### PR TITLE
samples: bluetooth: fast_pair: align sample name with new doc location

### PR DESCRIPTION
**Pull request by kapi-no**
Aligned names of Fast Pair samples with their new documentation location.

### CODE REVIEW

**Comment by kapi-no** [comment]
Follow-up change related to the following PR:

https://github.com/nrfconnect/sdk-nrf/pull/15531

The alternative is to call the sample category "Bluetooth Fast Pair sample" and leave the "Bluetooth" as the first word in the sample names.

**Comment by divipillai** [comment]
> Follow-up change related to the following PR:
> 
> #15531
> 
> The alternative is to call the sample category "Bluetooth Fast Pair samples" and leave the "Bluetooth" as the first word in the sample names.

I think we could call the sample category as "Bluetooth Fast Pair samples", like we do for the mesh samples. 

Also, the integration guide needs an update here : 
 (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/external_comp/bt_fast_pair.html#applications-and-samples)
Could link to the sample category.

**Comment by kapi-no** [comment]
> > Follow-up change related to the following PR:
> > #15531
> > The alternative is to call the sample category "Bluetooth Fast Pair samples" and leave the "Bluetooth" as the first word in the sample names.
> 
> I think we could call the sample category as "Bluetooth Fast Pair samples", like we do for the mesh samples.
> 
> Also, the integration guide needs an update here : (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/external_comp/bt_fast_pair.html#applications-and-samples) Could link to the sample category.

Okay, I suggest to align the integration guide in the follow-up PR as it will generate conflicts with the FMDN content for the integration guide.

**Comment by kapi-no** [comment]
@divipillai, aligned according to your suggestion

**Review by divipillai — APPROVED** [review]

**Review by alstrzebonski — APPROVED** [review]

File: doc/nrf/samples/fast_pair.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
-.. _fast_pair_samples:
+.. _bt_fast_pair_samples:
 
-Fast Pair samples
-#################
+Bluetooth Fast Pair samples
+###########################
```
Comment (divipillai) [inline doc/nrf/samples/fast_pair.rst:4]: Can we move the "samples/fast_pair" path after "samples/bl" in the "nrf\doc\nrf\sample" to follow the alphabetical order?

Comment (kapi-no) [reply doc/nrf/samples/fast_pair.rst:4]: Yes, sorry, missed that one

**Review by MarekPieta — APPROVED** [review]

**Review by divipillai — APPROVED** [review]

**Pull request merged**